### ISSUE TITLE
シミュレーションモデル検証の枠組み作り

### ISSUE THREAD

**Issue by rakawanegan** [rakawanegan/pbl-ssc_with_gfootball#1]

**Comment by rakawanegan** [comment]
コミット：b49fd19e403014f8e18b0748795f598a1f5dff94

![Image](https://github.com/user-attachments/assets/ac914053-9b5c-415b-a6e7-635daaa8ae6d)

**Comment by rakawanegan** [comment]
コード：https://github.com/rakawanegan/pbl-ssc_with_gfootball/blob/master/notebooks/bin_sim_or_real.ipynb

LightGBMで実装
```
params = {
    'objective': 'binary',
    'metric': 'auc',
    'boosting_type': 'gbdt',
    'n_estimators': 100,
    'learning_rate': 0.1,
    'num_leaves': 31,
    'max_depth': -1,
    'seed': 42,
    'n_jobs': -1,
    'verbose': -1,
}
```

### 選手情報をシャッフルしない場合
コミット：53af3c7f565c6e46f1b61d3d26e24c4444c6646c
Accuracy: 1.00
AUC: 1.00

<details><summary>
top-3 importance distribution
</summary>

![Image](https://github.com/user-attachments/assets/737db723-1356-4ac3-9b30-c7960bbbca68)

![Image](https://github.com/user-attachments/assets/487696d4-077d-42e8-b7ee-87f49a5cbffe)

![Image](https://github.com/user-attachments/assets/5955576f-a83d-4911-b746-574628b998c0)

</details>

### 選手情報をシャッフルする場合
コミット：2881540383269b27dc9536e97a09ae96ca42b2b2
Accuracy: 0.70
AUC: 0.79

<details><summary>
top-3 importance distribution
</summary>

![Image](https://github.com/user-attachments/assets/c7326388-28cb-4699-bbfe-7415cf573812)

![Image](https://github.com/user-attachments/assets/3c85badc-7890-40f0-a2c0-c4a70a2a0b26)

![Image](https://github.com/user-attachments/assets/0b6eeba8-e0f6-466c-a312-1ee653cf0955)

</details>
